File: Examples/AspNetMvcCS/Controllers/DocumentViewerController.Protection.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.AspNetMvcCS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult Protection()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 800,
                Height = 600,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.pdf",
                DeniedPermissions = DocumentViewerPermissions.Download 
                                    | DocumentViewerPermissions.DownloadAsPdf 
                                    | DocumentViewerPermissions.Print 
                                    | DocumentViewerPermissions.SelectText
            };

            return View(documentViewer);
        }
    }
}

File: Examples/AspNetMvcVB/Controllers/DocumentViewerController.Protection.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

	    Public Function Protection() As ActionResult
		    Dim documentViewer = New DocumentViewer() With {
			    .Width = 960,
			    .Height = 720,
			    .Resizable = True,
			    .Document = "~/App_Data/ExampleFiles/Default.pdf",
                .DeniedPermissions = DocumentViewerPermissions.Download _
                                     Or DocumentViewerPermissions.DownloadAsPdf _
	                                 Or DocumentViewerPermissions.Print _
                                     Or DocumentViewerPermissions.SelectText
		    }

		    Return View(documentViewer)
	    End Function

    End Class
End Namespace
